### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Somdotta07**
- Added HTML pages
- Added CSS files
- Added js code for dynamic cards
- Added images,button
- Build mobile menu
- Any suggestion is welcome
- Video will upload later(Please consider)

### CODE REVIEW

File: About.html [hunk 1]
```diff
@@ -4,10 +4,135 @@
...
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="about.css">
 </head>
 <body>
-    <script src="main.js"></script>
+    <!-- Header NAV BARS -->
+    <header class="header">
+        <!-- Only for Desktop -->
+         <div></div>
+        <div class="navbar1">
```
Comment (Mihndim2020) [inline About.html:16]: - [ ] For semantic purposes, all your navigation menus should be enclosed in a `<nav>` tag. Please kindly fix that.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @Somdotta07 , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Linters are passing
:heavy_check_mark: The project is running

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION



# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: About.html [hunk 2]
```diff
@@ -4,10 +4,136 @@
...
+            <div class="underlinefull"></div>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.  delectus!
+            </p>
+            <div class="Container">
+                <img src="media/pasteventtwo.jpg" class= "image1" alt="Past event1">
+                <div class="overlay1">Climbing World Cup 2020</div>
+                <div></div>
+                <img src="media/pasteventtwo.jpg"  class= "image2" alt="Past event2">
+                <div class="overlay2">Climbing World Cup 2018</div>
+            </div>
```
Comment (SajjadAhmad14) [inline About.html:99]: [ OPTIONAL ] Images are blur here and it's difficult to see overlay text on them. Kindly make the text and images clear.
images

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Somdotta Sarkar, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏
# OPTIONAL BUT RECOMMENDED
- [x] Please check inline comments.
## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**